esp/bindings/http/platform/httpservice.cpp

```diff
@@ -1136,6 +1136,30 @@ EspAuthState CEspHttpServer::authNewSession(EspAuthRequest& authReq, const char*
     return authSucceeded;
 }
 
+void CEspHttpServer::createGetSessionTimeoutResponse(StringBuffer& resp, ESPSerializationFormat format, IPropertyTree* sessionTree)
+{
+    __int64 timeoutAt = sessionTree ? sessionTree->getPropInt64(PropSessionTimeoutAt, 0) : 0;
```

Comment:
A general comment, I like to see more comments in the code.  In this instance, what is the meaning of 0 here?